enhancements/update/phased-rollouts.md

```diff
@@ -0,0 +1,316 @@
+---
+title: phased-update-rollouts
+authors:
+  - "@wking"
+reviewers:
+  - "@jottofar"
+  - "@LalatenduMohanty"
+  - "@sdodson"
+  - "@steveeJ"
+  - "@vrutkovs"
+approvers:
+  - TBD
+creation-date: 2020-07-20
+last-updated: 2020-09-22
+status: implementable
+---
+
+# Phased Update Rollouts
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement request proposes phased update rollouts, allowing release administrators to declare a time window over which update recommendations are phased into per-cluster requests for stable channel [Cincinnati graphs][cincinnati].
+
+## Motivation
+
+We [currently][channel-semantics] has `fast-4.y` and `stable-4.y` channels for each 4.y series.
+That allows users who want to hear about releases and related update edges immediately after we declare them supported to do so via `fast-4.y`, while users who want to wait and see if we still recommend the edge after its seen more volume can do so via `stable-4.y`, as discussed [here][fast-vs-stable].
```

Comment:
`That allows users who want to hear about releases and related update edges immediately after we declare them supported to do so via fast-4.y` , +1